
---
bool success
float64[] firstJointPositions
float64[] secondJointPositions